source/utilities.tex

```diff
@@ -4109,12 +4109,12 @@
 A program that instantiates the definition of \tcode{variant} with
 no template arguments is ill-formed.
 
-\rSec3[variant.ctor]{Constructors}
-
 \pnum
 In the descriptions that follow, let $i$ be in the range \range{0}{sizeof...(Types)},
```

Comment:
```suggestion
In the descriptions that of \ref{variant}, let $i$ be in the range \range{0}{sizeof...(Types)},
```
[variant.variant] is not enough. [variant.relops] and [variant.specalg], which come after it, says "for all _i_" without defining _i_. This suggestion would have it defined.

Reply:
perhaps
"In the descriptions <del>that</del> of \ref{variant}, let $i$ be in the range \range{0}{sizeof...(Types)}," 
?
Also, if we make this change, do we need the other suggested change ? This should cover that section too.

Reply:
> perhaps
> "In the descriptions ~that~ of \ref{variant}, let $i$ be in the range \range{0}{sizeof...(Types)},"
> ?

Definitely!

> Also, if we make this change, do we need the other suggested change ? This should cover that section too.

Which change are you talking about? I mentioned everything else because I don't think this would cover it.

Reply:
The other change in this pull request in [variant.helper] that elaborates on TI in variant_alternative. I believe your suggestion clarifies what TI means there too. On the other hand, "The Ith element of Types, where indexing is zero-based." reads much better than "The type TI". What do you think ?

Reply:
I think this is defining _i_ and `T`<sub>_i_</sub> specifically, and thus don't cover what `I` and `T`<sub>_`I`_</sub> might mean.

Reply:
review comments applied